### DISCUSSION TITLE
読み上げをユーザー名ではなく表示名にしたい

### DISCUSSION THREAD

**Discussion by cxxtxn** [ciffelia/koe#155] [Q&A] [answered]
読み上げをユーザー名(ID)ではなく、表示名(日本語)にしたいのですが、変更することは可能ですか？

**Comment by ciffelia** [comment]
現在はできません。時期は未定ですが、今後のアップデートで対応予定です。

**Reply by cxxtxn** [reply]
対応予定ありがとうございます。
ユーザーIDを辞書登録することで代用できることがわかったので、そちらで対応します。

**Comment by ciffelia** [comment, marked as answer]
[v2.0.0](https://github.com/ciffelia/koe/releases/tag/v2.0.0)で修正しました。
